### ISSUE TITLE
Problem with 'apt-get update' on Debian Wheezy (graylog 1.1)

### ISSUE THREAD

**Issue by unpiong** [Graylog2/fpm-recipes#38]
Hash problem for an 'apt-get update' on Debian Wheezy with graylog 1.1:

The 'Release' file for wheezy is not up to date (2015 mai 20):
packages.graylog2.org/repo/debian/dists/wheezy/Release

For instance, md5sum on 'Package' for amd64 don't match anymore:
f0238941874e0d087f9ea541f8c6949f  packages.graylog2.org/repo/debian/dists/wheezy/1.1/binary-amd64/Packages
(don't match MD5Sum from 'Release': de645ebbd0a3527cb3a773535f330a78 3451 1.1/binary-amd64/Packages)

**Comment by bernd** [comment]
I am unable to reproduce this on Debian wheezy, unfortunately. (see output below)

The date of the `dists/wheezy/Release` file is okay because it did not change after May 20. I am unable to find the checksums you posted.

Please make sure that you do not get old content from any proxies in between.

If you still see the problem, please post the output of the commands I executed below.

Thank you!

```
root@vagrant:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 7.8 (wheezy)
Release:	7.8
Codename:	wheezy

root@vagrant:~# apt-get install apt-transport-https
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  libcurl3-gnutls
The following NEW packages will be installed:
  apt-transport-https libcurl3-gnutls
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 431 kB of archives.
After this operation, 761 kB of additional disk space will be used.
Do you want to continue [Y/n]? y
WARNING: The following packages cannot be authenticated!
  libcurl3-gnutls apt-transport-https
Install these packages without verification [y/N]? y
Get:1 http://security.debian.org/ wheezy/updates/main libcurl3-gnutls amd64 7.26.0-1+wheezy13 [322 kB]
Get:2 http://http.us.debian.org/debian/ wheezy/main apt-transport-https amd64 0.9.7.9+deb7u7 [109 kB]
Fetched 431 kB in 5s (73.3 kB/s)                                                          
Selecting previously unselected package libcurl3-gnutls:amd64.
(Reading database ... 26490 files and directories currently installed.)
Unpacking libcurl3-gnutls:amd64 (from .../libcurl3-gnutls_7.26.0-1+wheezy13_amd64.deb) ...
Selecting previously unselected package apt-transport-https.
Unpacking apt-transport-https (from .../apt-transport-https_0.9.7.9+deb7u7_amd64.deb) ...
Setting up libcurl3-gnutls:amd64 (7.26.0-1+wheezy13) ...
Setting up apt-transport-https (0.9.7.9+deb7u7) ...

root@vagrant:~# dpkg -i graylog-1.1-repository-debian7_1.2.0-3_all.deb 
Selecting previously unselected package graylog-1.1-repository-debian7.
(Reading database ... 28088 files and directories currently installed.)
Unpacking graylog-1.1-repository-debian7 (from graylog-1.1-repository-debian7_1.2.0-3_all.deb) ...
Setting up graylog-1.1-repository-debian7 (1.2.0-3) ...

root@vagrant:~# apt-get update
Hit http://security.debian.org wheezy/updates Release.gpg                
Hit http://security.debian.org wheezy/updates Release                    
Hit http://http.us.debian.org wheezy Release.gpg
Hit http://security.debian.org wheezy/updates/main Sources
Hit http://http.us.debian.org wheezy-updates Release.gpg        
Hit http://security.debian.org wheezy/updates/contrib Sources
Hit http://security.debian.org wheezy/updates/main amd64 Packages
Hit http://http.us.debian.org wheezy Release                        
Hit http://security.debian.org wheezy/updates/contrib amd64 Packages
Hit http://security.debian.org wheezy/updates/contrib Translation-en
Hit http://http.us.debian.org wheezy-updates Release
Hit http://security.debian.org wheezy/updates/main Translation-en
Hit http://http.us.debian.org wheezy/main Sources   
Hit http://http.us.debian.org wheezy/main amd64 Packages
Hit http://http.us.debian.org wheezy/main Translation-en
Hit http://http.us.debian.org wheezy-updates/main Sources
Get:1 https://packages.graylog2.org wheezy Release.gpg [473 B]
Hit http://http.us.debian.org wheezy-updates/contrib Sources              
Hit http://http.us.debian.org wheezy-updates/main amd64 Packages/DiffIndex
Hit http://http.us.debian.org wheezy-updates/contrib amd64 Packages
Get:2 https://packages.graylog2.org wheezy Release [10.1 kB]
Hit http://http.us.debian.org wheezy-updates/contrib Translation-en           
Hit http://http.us.debian.org wheezy-updates/main Translation-en/DiffIndex
Get:3 https://packages.graylog2.org wheezy/1.1 amd64 Packages [5,618 B]
Ign https://packages.graylog2.org wheezy/1.1 Translation-en_US
Ign https://packages.graylog2.org wheezy/1.1 Translation-en
Fetched 16.2 kB in 12s (1,317 B/s)                                                                                                                            
Reading package lists... Done

root@vagrant:~# apt-get install graylog-server
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  graylog-server
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 75.8 MB of archives.
After this operation, 85.8 MB of additional disk space will be used.
Get:1 https://packages.graylog2.org/repo/debian/ wheezy/1.1 graylog-server all 1.1.3-1 [75.8 MB]
Fetched 75.8 MB in 1min 9s (1,093 kB/s)                                                                                                                       
Selecting previously unselected package graylog-server.
(Reading database ... 28092 files and directories currently installed.)
Unpacking graylog-server (from .../graylog-server_1.1.3-1_all.deb) ...
Setting up graylog-server (1.1.3-1) ...
Adding system user `graylog' (UID 106) ...
Adding new user `graylog' (UID 106) with group `graylog' ...
Not creating home directory `/var/lib/graylog-server'.
root@vagrant:~# 
```

**Comment by unpiong** [comment]
Thank you for your answer.
As you did, I was able to make an 'apt-get update' without error.

In fact, I am using a local mirror (from 'deb https://packages.graylog2.org/repo/debian wheezy 1.1') because my graylog server have no internet connection.
I made this local mirror for graylog 1.0 and everything was fine.
But now, I have to delete 'Release' and 'Release.gpg' if I want to install graylog 1.1 (without check) from my repository: it's not very secure.

I noticed that the 'Release' file on my mirror (packages.graylog2.org/repo/debian/dists/wheezy/Release) is dated 'mai 20'.
The 'Release' from 'apt-get update' without mirror (/var/lib/apt/lists/packages.graylog2.org_repo_debian_dists_wheezy_release) is dated 'june 19'.

But the 'Packages' file on my mirror (packages.graylog2.org/repo/debian/dists/wheezy/Release/1.1/binary-amd64/Packages) don't match either Release file ...
(but the md5sum inside this 'Packages' file match the '.deb' files)

To sum up, there is something wrong with my mirror but I can't point out the problem.

**Comment by joschi** [comment]
@unpiong Thanks for the update! I'm closing this issue now. Feel free to add another comment if another problem comes up.
